adoc/retail_chap_components.adoc

```diff
@@ -47,20 +47,23 @@ In this configuration, it may or may not provide DHCP and DNS services.
 [[retail.sect.components.pos-terminals]]
 === Point-of-Sale Terminals
 
-Point-of-sale (PoS) terminals can come in many different formats, such as cashier terminals, digital scales, or cash drawers.
+Point-of-sale (PoS) terminals can come in many different formats, such as cashier terminals, digital scales, or information kiosks.
 Every terminal, however, is provided by a vendor, who set some basic information about the device in the firmware.
 {smr} accesses this vendor information in order to determine how best to work with the terminal in use.
 In most cases, different terminals will require a different operating system (OS) image to ensure they work correctly.
-For example, a cashier terminal has a full graphical display, where a digital scale might only have a very basic text display.
+For example, an information kiosk has a high-resolution touchscreen, where a cashier terminal might only have a very basic display.
 While both of these terminals require similar processing and network functionality, they will require different OS images to ensure that the different display mechanisms work correctly.
 
+Note that {smr} works well for deployments containing up to around 5,000 terminals.
```

Comment:
Wondering about number separators.  Sometimes people use these separators only for numbers >9999.

Reply:
Hrm, good question. IBM says to omit the comma in four digit numbers unless needed 'for consistency', Chicago says to leave them in. Do we have style guidance on this one?

Reply:
> Hrm, good question. IBM says to omit the comma in four digit numbers unless needed 'for consistency', Chicago says to leave them in. Do we have style guidance on this one?

I think we don't have style guidance.  I grep'ed through our files:

```
bp_chap_systems_management.adoc:In the example above, we have assumed 500 current clients, so the soft limit is 5100, and the hard limit is 6000.
For example in the case of 5000 clients: `jabbersoftnofile5100 jabberhardnofile6000`
```

This seems to indicate that the comma in four digit numbers isn't needed 'for consistency'. ;)

Reply:
Sure, that's good enough for me. I'll take it out :)

Reply:
https://github.com/SUSE/doc-susemanager/pull/208